### ISSUE TITLE
Crash on start, caused by natural sort in plugins

### ISSUE THREAD

**Issue by skruppy** [nicolargo/glances#1696]
#### Description

Steps to reproduce:
- Have a network interface named `foo42` and one named `42foo`.
- Try to open glances
- Crash

The bug is caused by a too simple natural sort implementation in https://github.com/nicolargo/glances/blob/develop/glances/plugins/glances_plugin.py#L304, which can't compare things starting with a number against things not starting with a number.

#### Versions

* Glances & psutil (glances -V): Glances v3.1.4.1 with PsUtil v5.7.0
* Operating System (lsb_release -a): Gentoo

#### Logs

The actual output is messed up, here is a cleaned version:
```
Traceback (most recent call last):

File "/usr/lib/python-exec/python3.7/glances", line 11, in <module>
load_entry_point('Glances==3.1.4.1', 'console_scripts', 'glances')()

File "/usr/lib/python3.7/site-packages/glances/__init__.py", line 143, in main
start(config=config, args=args)

File "/usr/lib/python3.7/site-packages/glances/__init__.py", line 112, in start
mode.serve_forever()

File "/usr/lib/python3.7/site-packages/glances/standalone.py", line 151, in serve_forever
loop = self.__serve_forever()

File "/usr/lib/python3.7/site-packages/glances/standalone.py", line 138, in __serve_forever
ret = not self.screen.update(self.stats, duration=adapted_refresh)

File "/usr/lib/python3.7/site-packages/glances/outputs/glances_curses.py", line 962, in update
self.flush(stats, cs_status=cs_status)

File "/usr/lib/python3.7/site-packages/glances/outputs/glances_curses.py", line 937, in flush
self.display(stats, cs_status=cs_status)

File "/usr/lib/python3.7/site-packages/glances/outputs/glances_curses.py", line 515, in display
__stat_display = self.__get_stat_display(stats, layer=cs_status)

File "/usr/lib/python3.7/site-packages/glances/outputs/glances_curses.py", line 489, in __get_stat_display
max_width=plugin_max_width)

File "/usr/lib/python3.7/site-packages/glances/plugins/glances_plugin.py", line 797, in get_stats_display
'msgdict': self.msg_curse(args, max_width=max_width),

File "/usr/lib/python3.7/site-packages/glances/plugins/glances_network.py", line 289, in msg_curse
for i in self.sorted_stats():

File "/usr/lib/python3.7/site-packages/glances/plugins/glances_plugin.py", line 304, in sorted_stats
return sorted(self.stats, key=lambda stat: tuple(map(

TypeError: '<' not supported between instances of 'int' and 'str'
```

**Comment by RazCrimson** [comment]
@nicolargo I think the issue can be closed as it seems to be fixed in #1885

**Comment by skruppy** [comment]
Oh sorry, you are right. It works now (tested with 3.2.2). thanks.
